### PR TITLE
Make small fixes to GCP docs

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Review by loicmathieu — APPROVED** [review]
LGTM, thanks a lot.

File: docs/src/main/asciidoc/gcp-functions-http.adoc [hunk 1]
```diff
@@ -27,9 +27,9 @@ To complete this guide, you need:
 
 == Solution
 
-This guide walks you through running a Maven Archetype to generate a sample project then creating three http endpoints
-written with JAX-RS APIs, Servlet APIs or Vert.x Web APIs.  After building, you will then be able to deploy
-to Google Cloud.
+This guide walks you through generating a sample project followed by creating three http endpoints
```
Comment (gsmet) [inline docs/src/main/asciidoc/gcp-functions-http.adoc:30]:
```suggestion
This guide walks you through generating a sample project followed by creating three HTTP endpoints
```

File: docs/src/main/asciidoc/gcp-functions-http.adoc [hunk 2]
```diff
@@ -27,9 +27,9 @@ To complete this guide, you need:
...
+written with JAX-RS APIs, Servlet APIs or Vert.x Web APIs. Once built, you will then be able to deploy
```
Comment (gsmet) [inline docs/src/main/asciidoc/gcp-functions-http.adoc:31]:
```suggestion
written with JAX-RS APIs, Servlet APIs or Vert.x Web APIs. Once built, you will be able to deploy
```
I would drop the `then` here.

**Review by gsmet — APPROVED** [review]
Added two small comments but feel free to merge as is.

**Comment by geoand** [comment]
> Added two small comments but feel free to merge as is.

Thanks, applied!

**Pull request merged**